### ISSUE TITLE
Hangup after update to kernel  v3.16.7-ckt4-ev3dev1

### ISSUE THREAD

**Issue by BertLindeman** [ev3dev/ev3dev#252]
Started this in #243 and referred  #245 
Remark of @dlech in #243 
> Yes, what you are describing seems to be more related to that issue ( #245  ) 
and not this issue  ( #243 ).  Or perhaps a new issue.

*  Installed kernel v3.16.7-ckt4-ev3dev1

*  rebooted

* Configuration:
  in1: NXT sound sensor
  in4: NXT US sensor
  outA: PF leds

* Within minutes hangup.

* Last session lines:

```
09:14:06 root@ev3dev:~# journalctl -b 0 -k
Bus error
09:14:58 root@ev3dev:~#
```

* Hangs....

Will reboot with the same configuration and command to verify it's repeatable.

EDIT: 
* Not similar (No hang yet) `journalctl -b 0 -k` provides normal output
* port in4 with NXT US sensor not recognized on boot. nor on unplug/plug-in. 
  Well not recognized most of the times on re-plug. Does not seem related to which port.
  After a few times (3 or 4) plugging in each port in1-in4 only once success:

```
[Tue Jan 20 10:12:12 2015] lego-port port3: Removed device 'in4:nxt-i2c-host'
[Tue Jan 20 10:12:12 2015] lego-sensor sensor3: Unregistered
[Tue Jan 20 10:19:58 2015] lego-sensor sensor5: Bound to device '4-0001'
[Tue Jan 20 10:19:58 2015] i2c-legoev3 i2c-legoev3.4: registered on input port 2
[Tue Jan 20 10:19:58 2015] lego-port port1: Added new device 'in2:nxt-i2c-host'
```
Sorry for not having a repeatable scenario.

**Comment by BertLindeman** [comment]
Maybe this is a red herring.
While making a backup of my sd-card, I saw quite a number of I/O errors.
The last hang I had on *that* sd-card was doing something with journalctl.
So doing a lot of I/O. Could be some I/O did never finish? 
As no others reacted on this issue, I close this one as the hang did not re-occur here on another SD-card.
